release-notes.yml

```diff
@@ -7,6 +7,15 @@ description: >
   multi-tenancy environment (on premis vs. cloud hosted) is a question of a single line of config.
 
 releases:
+- version: 1.6.0
+  date: 2019-03-21
+  fixed:
+  - Missing tenant context in populated subdocuments ([#16](https://github.com/craftup/node-mongo-tenant/issues/16)).
+  changed:
+  - Models with tenant context (`Model.byTenant(...)`) will have a modified db connection object
+    returning Models bound to same context.
+  - Plugin instances do not have any longer a public reference to the schema they are created for. This is necessary to
```

Comment:
Should read: 
```yaml
-  Public schema reference of plugin instances became unenumerable.
   This is necessary to avoid inifinite loops in case two tenant aware schemas are used in a discriminator setup.
```

Reply:
Changed